tutoraspects/patches/openedx-dockerfile-post-python-requirements

```diff
@@ -1,4 +1,4 @@
 RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
-    pip install "platform-plugin-aspects==0.4.0"
+    pip install "git+https://github.com/openedx/platform-plugin-aspects.git@bmtcril/tracking_event_command"
```

Comment:
Needs updated when https://github.com/openedx/platform-plugin-aspects/pull/6 is released